Dockerfile

```
@@ -3,11 +3,27 @@ FROM python:3.12
 # TODO assert C.UTF8 and PYTHONUNBUFFERED are set correctly
 
 WORKDIR /srv
-RUN --mount=type=cache,target=/root/.cache pip install --upgrade pip-tools wheel
+
+COPY includes.sh .
+# hadolint ignore=SC2086
+RUN --mount=type=cache,target=/var/cache/apt \
+    rm /etc/apt/apt.conf.d/docker-clean \
```

```suggestion
﻿﻿    rm /etc/apt/apt.conf.d/docker-clean; echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache \
```
I believe you need to create apt conf to ensure they'll get cached